sql/moz-fx-data-shared-prod/telemetry_dev_cycle_external/expirements_stats_v1/query.py

```diff
@@ -0,0 +1,153 @@
+"""Experiment data downloaded from APIs, clean and upload to BigQuery."""
+
+import datetime
+import logging
+
+import click
+import requests
+from google.cloud import bigquery
+
+API_BASE_URL_EXPERIMENTS = "https://experimenter.services.mozilla.com"
+API_BASE_URL_METRIC_HUB = "https://github.com/mozilla/metric-hub/tree/main/jetstream"
+SCHEMA = [
+    bigquery.SchemaField("slug", "STRING"),
+    bigquery.SchemaField("start_date", "DATE"),
+    bigquery.SchemaField("enrollment_end_date", "DATE"),
+    bigquery.SchemaField("end_date", "DATE"),
+    bigquery.SchemaField("has_config", "BOOLEAN"),
+]
+DEFAULT_PROJECT_ID = "moz-fx-data-shared-prod"
+DEFAULT_DATASET_ID = "telemetry_dev_cycle_external"
+DEFAULT_TABLE_NAME = "experiments_stats_v1"
+
+
+def parse_unix_datetime_to_string(unix_string):
+    """Parse unix_string with milliseconds to date string."""
+    if not unix_string:
+        return None
+    return datetime.datetime.fromtimestamp(int(unix_string) // 1000).strftime(
+        "%Y-%m-%d"
+    )
+
+
+def get_api_response(url):
+    """Get json of response if the requests to the API was successful."""
+    response = requests.get(url)
+    response.raise_for_status()
+    return response.json()
+
+
+def store_data_in_bigquery(data, schema, destination_project, destination_table_id):
+    """Upload data to Bigquery in a single, non partitioned table."""
+    client = bigquery.Client(project=destination_project)
+
+    job_config = bigquery.LoadJobConfig(
+        create_disposition="CREATE_IF_NEEDED",
+        schema=schema,
+        source_format=bigquery.SourceFormat.NEWLINE_DELIMITED_JSON,
+        write_disposition="WRITE_TRUNCATE",
+    )
+
+    load_job = client.load_table_from_json(
+        data, destination_table_id, location="US", job_config=job_config
```

Comment:
Should we consider making location / region an argument with "US" being optional?

Reply:
if I move it out of my query.py yes, but not for a one off function?